go.mod

```diff
@@ -7,8 +7,9 @@ require (
 	github.com/onsi/ginkgo v1.12.0
 	github.com/onsi/gomega v1.9.0
 	github.com/spf13/afero v1.2.2
-	github.com/spf13/cobra v0.0.7
+	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/tools v0.0.0-20200403190813-44a64ad78b9b
+	golang.org/x/tools v0.0.0-20200616195046-dc31b401abb5
+	sigs.k8s.io/controller-tools v0.4.1 // indirect
```

Comment:
You should definetely not be bumping dependency versions in a doc PR (applies to `go.sum` too).

Reply:
Yeah, I was using ./docs/book/install_and_build.sh to test out the changes. Accidentally committed the go.mod/sum. Not intentional at all 😅 

Will un-commit these changes.